build/configs/imxrt1020-evk/imxrt1020-evk_download.sh

```diff
@@ -84,222 +43,49 @@ function imxrt1020_bootstrap()
 #Input: Address and length
 function flash_erase()
 {
-	echo -e "\nFLASH_ERASE: ADDR:$1 LENGTH:$2 KB"
-	size_in_bytes=$(($2 * 1024))
-	${BLHOST} -p ${TTYDEV} -- flash-erase-region $1 $size_in_bytes
+	echo -e "\nFLASH_ERASE: ADDR:$2 LENGTH:$3 KB"
+	size_in_bytes=$(($3 * 1024))
+	${BLHOST} -p $1 -- flash-erase-region $2 $size_in_bytes
 }
 
 ##Utility function to write a binary on a flash partition##
 #Input: Address and filepath
 function flash_write()
 {
-	echo -e "\nFLASH_WRITE ADDR:$1 \nFILEPATH:$2 "
-	${BLHOST} -p ${TTYDEV} -- write-memory $1 $2
+	echo -e "\nFLASH_WRITE ADDR:$2 \nFILEPATH:$3 "
+	${BLHOST} -p $1 -- write-memory $2 $3
 	sleep 2
 }
 
-##Utility function to match partition name to binary name##
-function get_executable_name()
-{
-	case $1 in
-		kernel) echo "tinyara.bin";;
-		micom) echo "micom";;
-		wifi) echo "wifi";;
-		zoneinfo) echo "zoneinfo.img";;
-		userfs) echo "imxrt1020-evk_smartfs.bin";;
-		*) echo "No Binary Match"
-		exit 1
-	esac
+
+# Treat adding the USB rule first
+function usb_rule(){
```

Comment:
I think ' { ' should be in next line ? Can you please check that ?

Reply:
Done.